parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -44,7 +44,11 @@ pub mod pallet {
 	}
 
 	#[pallet::event]
-	pub enum Event<T> {}
+	#[pallet::generate_deposit(pub(super) fn deposit_event)]
+	pub enum Event<T: Config> {
+		BeaconHeaderImported(H256, u64),
```

Comment:
More recent versions of Substrate allow you to name fields:
```suggestion
		BeaconHeaderImported { block_hash: H256, block_number: u64 },
```

Reply:
Much better, thanks
![Screenshot 2022-08-26 at 12 50 04](https://user-images.githubusercontent.com/1788149/186888214-f8093850-ec2a-4aa3-b172-a90007f3a7db.png)
!